rhp/v2/contracts.go

```diff
@@ -85,29 +85,27 @@ func ContractRenewalCollateral(fc types.FileContract, expectedNewStorage uint64,
 	if endHeight < blockHeight {
 		panic("current blockHeight should be lower than the endHeight")
 	}
-	duration := endHeight - blockHeight
 
-	// calculate the base collateral - if it exceeds MaxCollateral we can't add more collateral
+	// calculate the base collateral
 	baseCollateral := host.Collateral.Mul64(fc.Filesize).Mul64(extension)
+
+	// if it exceeds MaxCollateral we can't add more
 	if baseCollateral.Cmp(host.MaxCollateral) >= 0 {
 		return types.ZeroCurrency
 	}
 
-	// calculate the new collateral
-	newCollateral := host.Collateral.Mul64(expectedNewStorage).Mul64(duration)
```

Comment:
I slightly prefer `newCollateral` since it indicates that it's not the full collateral of the renewal but the counterpart of `baseCollateral`